book/website/reproducible-research/reviewing/reviewing-challenges.md

```diff
@@ -0,0 +1,122 @@
+Power imbalances that can lead to these situations should be avoided as far as possible by aligning processes and selecting reviewers in order to avoid this.
+However in many projects, especially open source projects, this may not be possible: core developers always have the final say on what gets merged.
+
+Where such power imbalances cannot be avoided it falls to the more senior contributor to act appropriately.
+For example, it can be helpful for them to use supportive rather than accusative language, to be more forgiving of others' language, and to take special care to encourage consensus rather than dictating an outcome.
+
+## The infinite wait
+
+Reviewing pull requests is time-consuming and can be seen as less productive than generating new content.
+In open source projects where there is no pre-existing relationship between the submitter of a pull request and the core team reviewing it, it can be easy for pull requests to get ignored.
+
+This is especially true for pull requests that are controversial, or which are of high quality but making changes that the project does not see as a priority internally.
+
+A common outcome is that a pull request will be left in the queue for months or even years without being addressed.
+This is almost guaranteed to alienated potential members of the community who might otherwise make more contributions.
+
+It's crucial that reviews are tackled in a timely manner.
+For an open source project, it is the responsibility of the project maintainers, and a sign of good stewardship, if pull requests are reviewed in good time.
+
+Allowing stale pull requests to build up can cause drag on a project.
+
+If a project isn't ready to accept a pull request or doesn't have the capacity to review it, it is beneficial to make this clear, either by rejecting the pull request, or by engaging with the author to explain the situation.
+
+## The infinite review
+
+If a reviewer doesn't like a piece of code, it's not unusual for them to simply leave a review incomplete, rather than tackle the thorny issues it raises.
+This may be especially the case if a reviewer is having difficulty articulating what they dislike about a proposed change.
+
+Leaving pull requests half reviewed, from the side of the author or reviewer, can feel disrespectful to other parties who have worked on it.
+
+If there is any uncertainty about the benefits or purpose of a particular change, it is better to make this clear through a comment, emphasising that you're looking to better understand.
+
+If a change is ultimately not going to be merged, it is more respectful to reject explicitly than to leave the review incomplete.
+When rejecting a change it is essential that the reason is explained for the benefit of the author.
+
+Remember that not everything has to be resolved in a single pull request.
+If it looks like it's going to be hard to come to a decision on something, create a new issue to deal with it, rather than allow it to block the code merge.
```

Comment:
actually this can happen from the contributors side as well. A change is reviewed and changes are requested and then the pr is stalling